client/src/locale/ru-RU.locale.ts

```diff
@@ -0,0 +1,134 @@
+import Locale from './locale'
+
+export default <Locale>{
+    home: {
+        title: 'Pacstall',
+        subtitle: 'AUR для Ubuntu',
+        cards: {
+            whyDifferent: {
+                title: 'Чем это отличается от любого другого пакетного менеджера?',
+                description:
+                    'Pacstall использует стабильную базу Ubuntu, но позволяет использовать ' +
+                    "bleeding-edge программы почти что без компромиссов, так что вам не " +
+                    'нужно беспокоиться о патчах безопасности или новом функционале.',
+            },
+            howItWorks: {
+                title: 'Тогда как это работает?',
+                description:
+                    ' Pacstall использует файлы, известные как <0>pacscript-ы</0> (похоже' +
+                    ' на PKGBUILD-ы) которые содержат обязательные компоненты для сборки пакетов,' +
+                    ' и собирает из них исполняемые файлы на вашей системе.',
+            },
+        },
+        installationInstructions: 'Инструкция по установке',
+        showcase: {
+            title: 'Showcase',
+            packageSearch: 'Package Search',
+        },
+    },
+    navbar: {
+        title: 'Pacstall',
+        contribute: {
+            title: 'Внести вклад',
+            workOnFeatures: 'Работать над новым функционалом',
+            helpTranslate: 'Помочь с переводом',
+            becomeAMaintainer: 'Станьте сопровождающим пакета',
+        },
+        social: {
+            title: 'Соцсети',
+            discord: 'Discord',
+            matrix: 'Matrix',
+            reddit: 'Reddit',
+            lemmy: 'Lemmy',
+            mastodon: 'Mastodon',
```

Comment:
Here.

Reply:
same goes for these, i can transliterate these, but these are names, and they're usually left as original ones in english

Reply:
Ok then keep those specifically.